examples/etcd/certs/ca-cert.pem

```diff
@@ -1,22 +1,139 @@
-LRQfLtYuhXS9uTNyScBhvMuqkTKbDJgvmihxvyDHUAdEnOYIXrza1XZ0aQNf9RdG
-3G16JHtKQWpOM2QwmHHQ3vzlklzih+N8D4a4/t/utadGMFYmK90lIUpr92DOKp6O
-O9PN0gN0kdVZsp/hslleMgscPCXLLLMOGB6lubFGyKWwZ0iAYyeSJhAADO6DSiHx
-8Jgg4Fq23f5hEArWAv1BgE4mdtKl35aHyY7szDxlYJYExMpx9ZnZAgMBAAGjUzBR
-MB0GA1UdDgQWBBSlBuNY0HzTUXe6nrQ/5PU8janvpjAfBgNVHSMEGDAWgBSlBuNY
-0HzTUXe6nrQ/5PU8janvpjAPBgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUA
-A4IBAQBX7yPdCiagT13D4kf5Rvx3CK/3aeoR5EZB0TuiJRPl+roAS/6wRRDTO1Kn
-8EW5vAFszzs8y3TNt9vvD714UBfBzk/AJXB5ih9NvqUf99/KoAYYYCXtHs/1qv2c
-Jx6Vj9uBQBBsMagVwSi6B8hFObfiXWOM2zbCjzMm3wtG8jo+ykN5fgCoVYYUfMNS
-IWehr8A928o5k0qosjfXciX7U/QoVO1awXI4NDjRlxFQAPrSDKy9BULIvsbi1//2
-fNZFnhwU6gmeq/OtCU+Qqduflz6qMTI9eBhg2lXcPmXkhyoNpYX+7vX4/JxYqznh
-556xKDkIMbv36lMGMa8vDmobabrG
+MIIGGTCCBAGgAwIBAgIJAOdzd4iKo0XjMA0GCSqGSIb3DQEBCwUAMGAxCzAJBgNV
+BAYTAlVTMRYwFAYDVQQIDA1TYW4gRnJhbmNpc2NvMQ0wCwYDVQQHDARTT01BMRYw
+FAYDVQQKDA1HcmF2aXRhdGlvbmFsMRIwEAYDVQQDDAlsb2NhbGhvc3QwHhcNMjIw
+OTE3MDAwODMzWhcNMzIwOTE0MDAwODMzWjBgMQswCQYDVQQGEwJVUzEWMBQGA1UE
+CAwNU2FuIEZyYW5jaXNjbzENMAsGA1UEBwwEU09NQTEWMBQGA1UECgwNR3Jhdml0
+YXRpb25hbDESMBAGA1UEAwwJbG9jYWxob3N0MIICIjANBgkqhkiG9w0BAQEFAAOC
+Ag8AMIICCgKCAgEA1N4FETACUqIz440VpaGgLOwZDd+VEvVuqXmSQJ0Zl8Xxy1lh
+rfgSQ/T3JzOTt25280f0wGddE7Y8qhpUzwmtwUo/nqGySibKLhzTL/eAQSXLUccF
+zoNDz02Myde8D/lcTsNdbBP0q+PuvB6XhQp+7NjApPji7xqBUzD3cYsPyuToNkmE
+x9dOdj55bc+tg9MjGMDbEKhzr13A5pPqrBZerbBOePL2Y2ONRcLvplsZrXA6w/V0
+qEFJt9ujFrSIJjJfoNFVWn1P7/loLQqxUXvZWgeScf5vMXWXG9UMQF9O6X2LcD39
+9s2X+6xNuWErqSEuN30g2hJG36SIL9dAmicYdEvAeiEr4Cpikc7UoXB7GI1tJ2GS
+EXSK9XyfPZNTy1LbcrmqFDFJO9xvaWHANsK3SSbvWR+bTQQpfHZeat+KQsbanYah
+T543Oedndexe5J3WRXFqhDfFLCC4LzSdFhXAVJMIvI/eZJCKiBaGbbRy0hBjDrcn
+bOJS3lgRbSDznj3pZcP5aZ1NnIl5jyaBws6glbGigIYY5pUcgMCfYWO2ThG4kR8A
+yFq9dX2i2p9AuXHjZwrLRbRodbCJBiOZQxSK9/Cx3UN6rHgWzkI/oz5M3FkI96hr
+q4GaTGk8bluKx8LeeFb3Q/P4HNAUziq3LAeobeh19YRSq16DQYWnm4udxksCAwEA
+AaOB1TCB0jAPBgNVHRMBAf8EBTADAQH/MAsGA1UdDwQEAwIBBjAdBgNVHQ4EFgQU
+jPoZ3feZNy7LUDCC7R9vVDwmtHgwgZIGA1UdIwSBijCBh4AUjPoZ3feZNy7LUDCC
+7R9vVDwmtHihZKRiMGAxCzAJBgNVBAYTAlVTMRYwFAYDVQQIDA1TYW4gRnJhbmNp
+c2NvMQ0wCwYDVQQHDARTT01BMRYwFAYDVQQKDA1HcmF2aXRhdGlvbmFsMRIwEAYD
+VQQDDAlsb2NhbGhvc3SCCQDnc3eIiqNF4zANBgkqhkiG9w0BAQsFAAOCAgEAU0EH
+cbRAOrTYG0jRiGtg+W0sJ8R50Kz7sgc1adx8XfFihhypMv0JtSVW4AVZi9Q2UJn/
+FYtDQ0YV1x4B4qEbOlo5GgrKYxLAfZtdVQvfyghUK3mnjaSfZxhhd4MGAVKbdvsG
+Pr1TkZW2ZtoTNx1zH8Xhr+UzM9B3og5XObROSCq2+IbsqtgibO5u3Sxm/6+cn8hg
+fSyz9BI2lB+sOwZqPtv8VeMrYKdZIDHXKXbHvNqylsGxJHwwYA3hd2M6XE+cCIp9
+3x4dkyLRLjYUIUP5+oWE5QHju+Dbsnj4uOBnaplHXoNw9wl+iDXFWQCMMrSVWG9j
+JtXcR0qXRlm1QIM63xg7gyCZHJ8CbdPOgkj57XfeuKrwX/J1BkeO+jqp44zqRn9r
+LQq+Hi7WSRBiLOOFOCiz5WJksqgFlA9aaGwED54D0MgDOPI5BOTotf2m5ZUuJMrA
+/NptccTXwPk2uN6F12DUPHrGjEDvvSN25AJXdPa2709/e63FbfcYc8tMF/rzNQPR
+Opiw9jIvhXW7KTiA38NofZvxdC0O/1S1PjVDWOwa2rw3qJErU/wZ+8kcQ8iZehtc
+eGUiGaPZ1wk4gQQaJWO7+1O0cwBhXtDMZ4hgW/UNLTqxlpQXHrsiNkPb16aJjmuq
+zNE+cHbpgRDe2ntfFgId6nUi29VOPcNtfWvNrsg=
+-----END CERTIFICATE-----
+Certificate:
```

Comment:
It doesn't look like a real problem, but what's up with the `-text` output being included here?

Reply:
@timothyb89 The `openssl ca` command includes text form of the certificate in the .pem file by default. It can be disabled with `-notext` flag but it's somewhat convenient to have a text form alongside pem. Most parsing tools e.g. Go's `pem.Decode()` handle it just fine.

Reply:
TIL, thanks!